src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -370,3 +401,22 @@ export function getSurvivalChartDataByAlteredStatus(
         patientToAnalysisGroups,
     };
 }
+
+export function generateSurvivalPlotTitleFromDisplayName(title: string) {
+    return title.replace(/status|survival/gi, '');
+}
+
+export function generateStudyViewSurvivalPlotTitle(title: string) {
+    let result = title.replace(/status/gi, '');
+    return result.toLowerCase().includes('survival')
+        ? result
+        : `${result} Survival`;
+}
+
+export function getSurvivalAttributes(clinicalAttribute: ClinicalAttribute[]) {
+    return _.chain(clinicalAttribute)
+        .map(attr => attr.clinicalAttributeId)
+        .filter(id => id.endsWith('_STATUS') || id.endsWith('_MONTHS'))
```

Comment:
you should use case insisitive regexpression.  /_STATUS$/i.test(...) instead of endsWith